addon/components/base/bs-form.js

```diff
@@ -261,6 +280,38 @@ export default Component.extend({
         isPresent(model) && isPresent(property)
       );
       set(model, property, value);
+    },
+
+    /**
+     * Submit action yielded in the bs-form template and accessible as `form.submit`
+     *
+     * @method actions.submit
+     * @param {Object} [model] Optional, falls back to model passed to bs-form
+     * @return {Promise} (not yet implemented)
+     * @public
+     */
+    submit(model=null) {
+      if(model) {
+        this.set('model', model);
+      }
+      return this.submit();
```

Comment:
Do we want to return a promise here that callers call use? Doing so could allow for some nice UI approaches (or wrapping these calls in ember-concurrency tasks)